### DISCUSSION TITLE
Freeplane not launching

### DISCUSSION THREAD

**Discussion by marcperuz** [freeplane/freeplane#1787] [Help]
**Describe the bug**
Freeplane is installed on Windows but I am unable to launch it

**To Reproduce**
Execute freeplane.bat. The output is 

```
C:\Program Files\Freeplane>java --add-exports=java.desktop/sun.awt.shell=ALL-UNNAMED --add-exports=java.desktop/com.sun.java.swing.plaf.windows=ALL-UNNAMED -Xmx1g -Dorg.freeplane.userfpdir="C:\Users\peruzzetto\AppData\Roaming\\Freeplane" -jar freeplanelauncher.jar
Picked up JAVA_TOOL_OPTIONS:  -Djava.vendor="Sun Microsystems Inc."
Unrecognized option: --add-exports=java.desktop/sun.awt.shell=ALL-UNNAMED
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

**Expected behavior**
Freeplane starts

**Desktop (please complete the following information):**
 - OS: Windows 
 - Version 1.11.11
 - Java Version (output of java -version)
```
Picked up JAVA_TOOL_OPTIONS:  -Djava.vendor="Sun Microsystems Inc."
java version "1.8.0_271"
Java(TM) SE Runtime Environment (build 1.8.0_271-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.271-b09, mixed mode)
```

**Comment by yasenstar** [comment]
Hi Marc,

I'm using Windows 11 with following OpenJDK 11, it's no problem running the freeplace.exe.

openjdk 11.0.16.1 2022-08-12 LTS
OpenJDK Runtime Environment Microsoft-40648 (build 11.0.16.1+1-LTS)
OpenJDK 64-Bit Server VM Microsoft-40648 (build 11.0.16.1+1-LTS, mixed mode)

Possible you can try this alternative JDK and see. Good Luck!

Xiaoqi

**Comment by albert-yau-sp** [comment]
Same issue here.  Have installed the openjdk alternative and it didn't help

**Comment by dpolivaev** [comment]
freeplane.bat is not smart enough to handle differences between Java versions. Use freeplane.exe or freeplaneConsole.exe

**Comment by dpolivaev** [comment]
Actually, freeplane.bat is only added for people who want to play with different Java options and know exactly what they do.

**Comment by marcperuz** [comment]
Hello. Sorry for the long delay of my answer. I actually ran freeplane.bat following the "troubleshooting" recommandations here, as Freeplane was not launching (https://docs.freeplane.org/attic/old-mediawiki-content/Troubleshooting.html). When I execute freeplane.exe or freeplaneConsole.exe, nothing happens either. 

I tried using openjdk, but it does not solve the problem.

**Comment by dpolivaev** [comment]
Post here the output from freeplaneConsole.exe

**Comment by marcperuz** [comment]
There is no output, just a blank line.

**Comment by dpolivaev** [comment]
No clue

**Comment by marcperuz** [comment]
It actually now works. I think the envrionment variables were not properly updated after I installed the latest version of Java JRE. I also deleted the JAVA_TOOL_OPTIONS environment variable, it may have solved the problem.
